Learning/eval_policy.py

```diff
@@ -137,39 +191,98 @@ def evaluate_baseline_policy_multi(env, policy, collection_time_steps, sector_ro
     metrics = env.get_metrics()
     save_data(metrics, trial, save_dir)
     
-def evaluate_baseline_policy_serial(env, policy, collection_time_steps, sector_rows, sector_cols, 
+def evaluate_analytic_policy_serial(wrapper_sel, env, policy, collection_time_steps, sector_rows, sector_cols, 
```

Comment:
Nice.  Let's make wrapper_sel the third parameter for this method to keep consistency w/ the other eval method signatures.